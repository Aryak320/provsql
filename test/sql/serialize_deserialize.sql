\set ECHO none
SET search_path TO provsql_test,provsql;

select dump_data();
select read_data_dump();