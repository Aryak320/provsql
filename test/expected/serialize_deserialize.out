\set ECHO none
INFO:  serializing completed without error
 dump_data 
-----------
 
(1 row)

INFO:  deserialization completed without error
 read_data_dump 
----------------
 
(1 row)

